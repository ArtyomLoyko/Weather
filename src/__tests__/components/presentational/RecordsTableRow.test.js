import React from 'react';
import { shallow } from 'enzyme';
import RecordsTableRow from './../../../components/presentational/RecordsTableRow';

describe('RecordsTableRow', () => {
    const props = {
        username: 'alex',
        email: 'alex@gmail.com',
        score: 25,
    };

    const recordsTableRow = shallow(<RecordsTableRow {...props} />);

    it('renders properly', () => {
        expect(recordsTableRow).toMatchSnapshot();
    });
});
