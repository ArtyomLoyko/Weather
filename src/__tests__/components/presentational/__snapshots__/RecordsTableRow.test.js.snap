// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordsTableRow renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RecordsTableRow
    email="alex@gmail.com"
    score={25}
    username="alex"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <td>
          alex
</td>,
        <td>
          alex@gmail.com
</td>,
        <td>
          25
</td>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "alex",
        },
        "ref": null,
        "rendered": "alex",
        "type": "td",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "alex@gmail.com",
        },
        "ref": null,
        "rendered": "alex@gmail.com",
        "type": "td",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 25,
        },
        "ref": null,
        "rendered": 25,
        "type": "td",
      },
    ],
    "type": "tr",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <td>
            alex
</td>,
          <td>
            alex@gmail.com
</td>,
          <td>
            25
</td>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "alex",
          },
          "ref": null,
          "rendered": "alex",
          "type": "td",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "alex@gmail.com",
          },
          "ref": null,
          "rendered": "alex@gmail.com",
          "type": "td",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 25,
          },
          "ref": null,
          "rendered": 25,
          "type": "td",
        },
      ],
      "type": "tr",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
